pkl-cli/pkl-cli.gradle.kts

```diff
@@ -179,7 +190,7 @@ fun Exec.configureExecutable(
   executable = "${graalVm.baseDir}/bin/$nativeImageCommandName"
 
   // JARs to exclude from the class path for the native-image build.
-  val exclusions = listOf(libs.truffleApi, libs.graalSdk).map { it.get().module.name }
+  val exclusions = listOf(libs.graalSdk).map { it.get().module.name }
```

Comment:
the SDK is a virtual artifact anyway and is ignored by `native-image`; `truffleApi` _must_ be on the modulepath due to the operating mechanism behind Truffle Unchained. otherwise, build errors surface

this is also why non-relocations must be added for truffle types